runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningConfig.java

```diff
@@ -53,4 +59,76 @@ default boolean showMiningStats()
 	{
 		return true;
 	}
+
+	@ConfigSection(
+		name = "Graphics Settings",
+		description = "Overlay Graphics settings",
+		position = 10
+	)
+	String visualSection = "Graphics";
+
+	@Alpha
+	@ConfigItem(
+		position = 11,
+		keyName = "hexColorRockOverlay",
+		name = "Overlay color",
+		description = "Color of the rock overlay",
+		section = visualSection
+	)
+	default Color getRockOverlayColor()
+	{
+		return Color.YELLOW;
+	}
+
+	@Alpha
+	@ConfigItem(
+		position = 12,
+		keyName = "hexColorRockOverlayOutline",
+		name = "Overlay outline color",
+		description = "Color of the rock outline overlay",
+		section = visualSection
+	)
+	default Color getRockOverlayOutlineColor()
+	{
+		return Color.ORANGE;
+	}
+
+	@Alpha
+	@ConfigItem(
+		position = 13,
+		keyName = "hexColorRockOverlayMLM",
+		name = "MLM Overlay color",
```

Comment:
This is used for more than just MLM, but instead anything which can respawn. It is used for showing that enough time has elapsed that it is possible it can respawn. Additionally the default values for these should match the existing values we use so that the default behavior doesn't change.